### DISCUSSION TITLE
Too many log in attempts.

### DISCUSSION THREAD

**Discussion by LaMarrash** [runelite/runelite#16800] [Q&A]
I can log in in my phone, but through computer it says "Too Many Log In Attempts...". The main issue is that I'm on Linux, and the solution for this is downloading "Jagex Installet" and this software is not available on LInux

**Comment by Felanbird** [comment]
This problem is unfortunately not something we can help with, this is a restriction implemented by Jagex and using their launcher to start Runelite is the only known workaround, other than waiting until the issue goes away.
